### ISSUE TITLE
VO description stops once object with focus highlights is reached

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/balloons-and-static-electricity#538]
**Test device**
MacBook Air (m1 chip)

**Operating System**
11.5.2

**Browser**
safari

**Problem description**
https://github.com/phetsims/qa/issues/706

When using VO, if I press VO keys + B to read through the entire description, the description will stop being read once it reaches the object in the play area that has focus highlights. Is that correct behavior? I also saw this same behavior in  published sims (Molarity). 

**Steps to reproduce**

Here is an example:
1. Turn on VO
2. Use keyboard nav to select 2 balloons
3. Tab to the yellow balloon
4. Press VO + B to read description from the beginning. It will stop reading once it gives details about the yellow balloon in the play area. It will not move on to details about the green balloon, the wall, etc. If the focus had been on the green balloon, the description would have continued and read through details about it and then stopped.

**Visuals**

https://user-images.githubusercontent.com/87318828/133289040-d590a3ef-11fd-412e-9029-195564e62747.mov

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Balloons and Static Electricity‬
URL: https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.5.0-rc.1/phet/balloons-and-static-electricity_all_phet.html
Version: 1.5.0-rc.1 2021-09-07 23:14:39 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/93.0.4577.63 Safari/537.36
Language: en-US
Window: 1351x659
Pixel Ratio: 2/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 80)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

**Comment by jbphet** [comment]
I don't know enough about description to say whether the behavior is correct.  Assigning to @jessegreenberg for comment.

**Comment by jbphet** [comment]
Reviewed with @emily-phet, and we don't consider this blocking for the 1.5 release.

**Comment by jessegreenberg** [comment]
This does not seem like the correct behavior, but it is also not something we can control. Ill try it out outside of a PhET sim and see if it is happening there.

**Comment by jessegreenberg** [comment]
I am not seeing VO do this at http://htmlpreview.github.io/?https://github.com/phetsims/a11y-research/blob/master/reader-intro.html.

**Comment by jessegreenberg** [comment]
I came back to BASE and I am no longer seeing this happen in VO.

**Comment by jessegreenberg** [comment]
I just tried this several more times. A couple of times VO stopped on "heading level 1, Balloons and Static Electricity". Other times it read through everything, other times it got stuck on the balloon.

**Comment by jessegreenberg** [comment]
Ah, I just saw this happen in a basic HTML page at http://htmlpreview.github.io/?https://github.com/phetsims/a11y-research/blob/master/reader-intro.html, it stopped at the "Click Me!" button.

**Comment by jessegreenberg** [comment]
Oh! It is not a bug. According to https://www.apple.com/voiceover/info/guide/_1131.html VO + B will "Read from the beginning of a webpage to the current location". So if VO+B is pressed while focus is on one of the balloons, it will stop reading there.

@Nancy-Salpepi is VO+B documented anywhere in QA resources, should it be updated to reflect this? If so, can you point me to where? Otherwise this can be closed.

**Comment by Nancy-Salpepi** [comment]
@jessegreenberg 
I got VO + B from the document that @terracoda shared with me titled: Tip Sheet--Using VoiceOver with a PhET sim. I would ask her about it.

**Comment by jessegreenberg** [comment]
OK thanks @Nancy-Salpepi. @terracoda, assigning to you to decide if anything in that tip sheet needs to change. Otherwise, I think this can be closed.

**Comment by terracoda** [comment]
Thank you @jessegreenberg and @Nancy-Salpepi for investigating the issue. I have always wondered why VO stops randomly when reading from the top, or read straight through to the end.

I will update the Tip Sheet. VO stops where focus is. If no focus is set it will read through everything to the end.

I updated the Tip Sheet for the description course.

**Comment by terracoda** [comment]
Closing issue - nothing to fix here.
